content/docs/concepts/asm/_index.md

```diff
@@ -0,0 +1,276 @@
+
+Elasticity in supply and demand for Auton is absorbed by dynamically adjusting CDP incentives to increase and decrease Auton borrowing costs when Auton price moves above or below its Stabilization Target the [Auton Currency Unit (ACU)](/glossary/#acu).
+
+## ASM identifiers and accounts
+
+The ASM functions with two identities for cryptographic security: the CDP owner and the stabilization protocol contract.
+
+Stabilization Contract calls to mint and burn Auton as CDP's are interacted with to borrow and repay Auton are  restricted to the Stabilization Contract address, the '`stabilizer`' protocol address.
+
+### CDP identifiers
+
+The CDP owner's account address is used as a unique identifier for the CDP itself as well as the CDP owner.
+
+The identity is in the form of an ethereum format account address and is used to:
+
+- unambiguously identify the CDP on-chain
+- by the CDP owner as the `msg.sender` address in all CDP interactions: to open, repay debt, and withdraw collateral from the CDP
+- by a Liquidator to identify the CDP being liquidated in a liquidation scenario.
+
+
+## Stabilization protocol
+
+ASM roles, core concepts, and the lifecycle of a CDP from opening to closure.
+
+### Stabilization roles
+
+There are three roles in the ASM:
+
+- *Borrower (CDP owner)*: a user taking out a CDP to borrow Auton, deposing collateral in return. There is no limit on the number of open CDP's a _borrower_ can own at any one time. Borrower's must have an [account](/glossary/#account) on the Autonity network.
+- *Liquidator (Keeper)*: a user or agent that liquidates a CDP that has brobecome under collateralized, repaying the CDP's outstanding debt and receiving remaining collateral token in return. Liquidator's must have an [account](/glossary/#account) on the Autonity network.
+- *Stabilizer (ASM Protocol)*: the `stabilizer` is the protocol account address used for Auton `mint` and `burn` operations by the Stabilization Contract)
+
+### ASM elements
+
+The ASM is composed of 3 system elements implemented as smart contract logic: ACU, Supply Control, Stabilization. ASM contracts are deployed by the protocol at genesis.
+
+#### ACU
+
+The [Auton Currency Unit (ACU)](/glossary/#acu) is a currency basket from which an index value is computed. This index value is then used as the _stabilization target_ for Auton price.
+
+The ACU currency basket is composed of 7 free-floating currencies:
+
+- AUD - Australian Dollar
+- CAD - Canadian Dollar
+- EUR - Euro
+- GBP - British Pound Sterling
+- JPY - Japanese Yen
+- SEK - Swedish Krona
+- USD - United States Dollar
+
+Each currency's quantity in the basket is computed to provide a currency basket with minimal variance. The index value then has minimal volatility with respect to variance from individual currency fluctuations.
```

Comment:
Each currency's quantity in the basket is computed to provide a currency basket with minimal **total** variance **with respect to its underlying currencies**. The index value then has minimal volatility with respect to variance from individual currency fluctuations.

Reply:
@cmjc For ACU underlying currency quantity computation, it is computed through an optimization process, which aims to minimize the average of all currencies' variance w.r.t. the basket (the variance of the basket itself is not optimized).

In other words, it aims to minimize `AVERAGE(VAR(AUD/ACU), VAR(CAD/ACU), VAR(EUR/ACU), VAR(GBP/ACU), VAR(JPY/ACU), VAR(USD/ACU), VAR(SEK/ACU))`.

Reply:
Done in commit 6240f03709feec9b14e779a078f0624f50b1aa7d